apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -92,38 +90,26 @@
   private static final Logger LOGGER_RECORD_COUNTS =
       LoggerFactory.getLogger(RifLoader.class.getName() + ".recordCounts");
 
-  private final MetricRegistry appMetrics;
   private final LoadAppOptions options;
-  private final HikariDataSource dataSource;
-  private final EntityManagerFactory entityManagerFactory;
   private final IdHasher idHasher;
+  private final PipelineApplicationState appState;
+  private final SecretKeyFactory secretKeyFactory;
```

Comment:
This field is not used anywhere in this class.

Reply:
Hmm.  Git is showing shows two lines above the ocmment - meant for it just to show `secretKeyFactory`.

Reply:
Good catch -- I goofed this in the merge conflict resolution. Fixed.